django/core/handlers/asgi.py

```diff
@@ -0,0 +1,325 @@
+import asyncio
+import cgi
+import codecs
+import logging
+import sys
+import tempfile
+import traceback
+from io import BytesIO
+
+from asgiref.sync import sync_to_async
+
+from django import http
+from django.conf import settings
+from django.core import signals
+from django.core.exceptions import RequestAborted, RequestDataTooBig
+from django.core.handlers import base
+from django.http import FileResponse, HttpResponse, HttpResponseServerError
```

Comment:
Can we import also `HttpRequest`, `HttpResponseBadRequest`, `parse_cookie`, and `QueryDict` and remove `from django import http`?

Reply:
:+1: